be/src/util/threadpool.h

```diff
@@ -156,6 +157,12 @@ class ThreadPoolBuilder {
 //    thread_pool->SubmitFunc(std::bind(&Func, 10));
 class ThreadPool {
 public:
+    enum Priority {
+        LOW_PRIORITY,
```

Comment:
```suggestion
        LOW_PRIORITY = 0,
```

Reply:
0 is the default starting value.
https://stackoverflow.com/questions/34811486/do-c-enums-start-at-0

Reply:
sure, I'd like make it clear, easy to understand

Reply:
updated